buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -454,31 +455,33 @@ final PoolThreadCache threadCache() {
         return threadCache.get();
     }
 
-    // Too noisy at the moment.
-    //
-    //public String toString() {
-    //    StringBuilder buf = new StringBuilder();
-    //    int heapArenasLen = heapArenas == null ? 0 : heapArenas.length;
-    //    buf.append(heapArenasLen);
-    //    buf.append(" heap arena(s):");
-    //    buf.append(StringUtil.NEWLINE);
-    //    if (heapArenasLen > 0) {
-    //        for (PoolArena<byte[]> a: heapArenas) {
-    //            buf.append(a);
-    //        }
-    //    }
-    //
-    //    int directArenasLen = directArenas == null ? 0 : directArenas.length;
-    //
-    //    buf.append(directArenasLen);
-    //    buf.append(" direct arena(s):");
-    //    buf.append(StringUtil.NEWLINE);
-    //    if (directArenasLen > 0) {
-    //        for (PoolArena<ByteBuffer> a: directArenas) {
-    //            buf.append(a);
-    //        }
-    //    }
-    //
-    //    return buf.toString();
-    //}
+    /**
+     * Returns the status of the allocator (which contains all metrics) as string. Be aware this may be expensive
+     * and so should not called too frequently.
+     */
+    public String dumpStatus() {
```

Comment:
Sounds good

> Am 09.04.2016 um 15:35 schrieb Trustin Lee <notifications@github.com>:
> 
> In buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java:
> 
> > -    //    buf.append(directArenasLen);
> > -    //    buf.append(" direct arena(s):");
> > -    //    buf.append(StringUtil.NEWLINE);
> > -    //    if (directArenasLen > 0) {
> > -    //        for (PoolArena<ByteBuffer> a: directArenas) {
> > -    //            buf.append(a);
> > -    //        }
> > -    //    }
> > -    //
> > -    //    return buf.toString();
> > -    //}
> > +    /**
> > +     * Returns the status of the allocator (which contains all metrics) as string. Be aware this may be expensive
> > +     * and so should not called too frequently.
> > +     */
> > +    public String dumpStatus() {
> dumpStats()?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>